addons/sale_timesheet/models/project_overview.py

```diff
@@ -92,7 +92,8 @@ def _plan_prepare_values(self):
         other_revenues = self.env['account.analytic.line'].read_group([
             ('account_id', 'in', self.analytic_account_id.ids),
             ('amount', '>=', 0),
-            ('project_id', '=', False)], ['amount'], [])[0].get('amount', 0)
+            ('project_id', '=', self.id),
+            ('so_line', '=', False)], ['amount'], [])[0].get('amount', 0)
```

Comment:
Hello,
After checking with AJU, analytic lines which are not linked to the project (but part of the account) should also appear in "Other revenues".

```suggestion
            ('project_id', '=', False)
            ('so_line', '=', False)], ['amount'], [])[0].get('amount', 0)
```
What about that ? Would it solve the issue ?

Reply:
@LucasLefevre No it doesn't solve the issue. With your suggestion we just have the initial issue.
As explained in my commit message, when validating the invoice I, an analytic line L1 is created and L1 is not linked to the project PR. In the overview, L1 is counted twice once in the invoiced part and once in the Other revenue (because L1 is not linked to PR and not linked to a SO line. Perhaps do we have to link L1 to PR when validating I and keep the domain as it was initially?